pyengine/katana.py

```diff
@@ -33,9 +33,10 @@ def matcher(qty,book):
 
             ordqty = bqqty # in a normal case we use the full quote
```

Comment:
does this still work if we unindent this block? i think we are unnecessarily evaluating this for every quote, we only need to do it for the ones we use

Reply:
yes it looks like it evaluates it for every quote. unindenting the block wont work but actually if we move it into the "if price <= lowestprice:" block it will only evaluate for the lowest quote (at the time)

ie if the order is 3 2 1
3 - evaluate
2 - evaluate
1 - evaluate

but if its 2 3 1
2 - evaluate
3 - skip
1 - evaluate

rly simple to fix this way, prob need to do some rewriting to be maximally efficient

Reply:
simple way seems fine. mebbe i look at rewrite.

Reply:
change looks good